src/RegistryCoordinator.sol

```diff
@@ -515,7 +515,18 @@ contract RegistryCoordinator is
                 OperatorInfo({operatorId: operatorId, status: OperatorStatus.REGISTERED});
 
             // Register the operator with the EigenLayer core contracts via this AVS's ServiceManager
-            serviceManager.registerOperatorToAVS(operator, operatorSignature);
+            bool operatorSetAVS = IAVSDirectory(serviceManager.avsDirectory()).isOperatorSetAVS(address(serviceManager)); /// TODO: call avsdirectory
+            if (operatorSetAVS){
+                bytes memory quorumBytes = BitmapUtils.bitmapToBytesArray(quorumsToAdd);
+                uint32[] memory operatorSetIds = new uint32[](quorumBytes.length);
+                for (uint256 i = 0; i < quorumBytes.length; i++) {
+                    operatorSetIds[i] = uint8(quorumBytes[i]);
```

Comment:
Why `uint8(quorumBytes[I])`?

Reply:
We can't go directly from bytes1 -> uint32

the uint8 to uint32 happens implicitly when passed to the function call